src/jit/lclvars.cpp

```diff
@@ -262,6 +264,9 @@ void Compiler::lvaInitTypeRef()
         }
     }
 
+    info.compArgRegCount = argRegNum;
```

Comment:
This doesn't make sense to me. Why are you assigning a register number to a count?